.evergreen/config.yml

```diff
@@ -598,6 +598,18 @@ tasks:
             SKIP_CRYPT_SHARED: "yes"
             TESTS: "csfle"
 
+    - name: "test-without_aws_credentials"
+      commands:
+        - func: "bootstrap mongo-orchestration"
+          vars:
+            TOPOLOGY: "replica_set"
+        - func: "start kms servers"
+        - func: "run tests"
+          vars:
```

Comment:
I just improved my understanding of the `vars` construct by reading through [Expansions](https://github.com/evergreen-ci/evergreen/wiki/Project-Configuration-Files#expansions) in the Evergreen docs.

> Notice that the function reference can define a set of vars which are treated as expansions within the configuration of the commands in the function.

I reckon my prior confusion was due to our historically inconsistent naming/formatting. I assumed some (like "TESTS") were environment variables, but that's not the case until we explicitly export or declare them when invoking a shell command.

A few more of these PRs and I'll probably want to take on some Evergreen config refactoring.

Reply:
> A few more of these PRs and I'll probably want to take on some Evergreen config refactoring.

I'll make a note of it to remember when the next refactoring inevitably comes around...